specification/keyvault/resource-manager/readme.python.md

```diff
@@ -21,10 +21,21 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python) && $(multiapi)
 batch:
+  - tag: package-2019-09
```

Comment:
Changes look good, but I wonder if this is only stable, or preview as well? We're about to introduce a 2020-01-23 preview for control plane.

Reply:
i got release request for this one specifically